.github/workflows/dependabot-auto-merge.yml

```diff
@@ -0,0 +1,20 @@
+##
+ # Dependabot auto-merge config for merging patch and minor updates.
+ #
+ # Based on https://docs.github.com/en/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions#enable-auto-merge-on-a-pull-request
+ # ______________________________________________________________________________
+ #
+ # Maintained by brightSPARK Labs.
+ # www.brightsparklabs.com
+ ##
+
+name: Dependabot auto-merge
+on: [pull_request]
+
+jobs:
+  call-gradle-workflow:
+    uses: brightsparklabs/github-actions/.github/workflows/java.yml@master
```

Comment:
This should call the `gradle-plugins.yml` workflow instead of the `java.yml` workflow.